test_webhook.py

```
@@ -0,0 +1,80 @@
+        try:
+            response_body = response.json()
+        except:
+            response_body = response.text
+        print(f"Record start: {response.status_code} - {response_body}")
+    except Exception as e:
+        print(f"❌ Record start failed: {e}")
+        return False
+    
+    # Test record stop endpoint
+    try:
+        response = requests.post(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)